packages/storybook/stories/toggletip.mdx

```diff
@@ -0,0 +1,64 @@
+import { Toggletip } from '@equisoft/design-elements-react';
+import { ArgTypes, Meta, Canvas } from '@storybook/blocks';
+import * as ToggletipStories  from './toggletip.stories';
+
+<Meta of={ToggletipStories} />
+
+# Toggletip
+
+## Table of contents
+1. [Definition](#definition)
+2. [Usage](#usage)
+3. [Variants](#variants)
+4. [Properties](#properties)
+
+## Definition
+Toggletips are used to display additional information about a target element, without the user losing the context of
+the page [2].
+<Canvas of={ToggletipStories.Default} />
+
+## Usage
+
+### When to use
+- Use sparingly and only when the information cannot be presented to the user in any other way (e.g. directly on the
+page)[7].
+- Provides additional information about a page element (feature, action, content, etc) [1].
+- Use to display instructions or tips that help the user interact with the user interface more effectively.
+- Use to provide brief, useful content.
+- Use for secondary information [4]. Use it when it is not essential for the user to act on the information or complete
+  a task [2].
+- Use when you don‘t want to let the user lose context, but still want to provide additional useful unique information
+  about referring elements. [2].
+- Highlight a new feature or important feature that extends or improves the user's experience on the page [8].
```

Comment:
Question de garder la même formulation que les autres points. Commence la phrase avec "Use to..."
> Use to highlight...